Sources/XMLCoder/Auxiliaries/XMLCoderElement.swift

```diff
@@ -348,30 +350,30 @@ extension XMLCoderElement {
         self.init(key: key, elements: elements, attributes: attributes)
     }
 
-    init(key: String, box: SimpleBox) {
+    init(key: String, box: SimpleBox, attributes: [Attribute] = []) {
```

Comment:
I think you can do without the attributes in the `SimpleBox` initializer!

Reply:
yes yes nice catch 👍